jobs/fatal_od.py

```diff
@@ -40,8 +40,22 @@ def combine_date_and_time(self, in_data):
         )
         return
 
+    @post_load
```

Comment:
This is another issue that came up that I forgot about.  When loading the data the upsert request it gets serialized to json.  However, I kept getting the exception that the `datetime.datetime` is not JSON serializable. I add this fix int eh police blotter schema but simply making a string out of the combination instead of adding a separate method.